jetbrains-core/src/software/aws/toolkits/jetbrains/ui/connection/AwsConnectionSettings.form

```diff
@@ -42,9 +42,7 @@
       </component>
       <vspacer id="c7f64">
         <constraints>
-          <grid row="2" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false">
-            <preferred-size width="91" height="14"/>
```

Comment:
What was this preferred size doing before?

Reply:
Unsure, not sure it actually did anything